visualization-project-laptop-sales/visualization_project_laptop_sales.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+"""visualization-project-laptop-sales.ipynb
+
+Automatically generated by Colab.
+
+Original file is located at
+    https://colab.research.google.com/drive/11PjJHVqauhHbRPFK7HyT3EJIKPWtX2OS
+"""
+
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+df = pd.read_csv("/content/Sales_data.txt")
+
+df
+
+df.info()
+
+#Question 1: What is target market? Male or female?
+#The mean profit from females are 148.38
+#while the mean profit for males is 133.47
+
+df.groupby('Contact Sex')['Profit'].mean()
+
+df.groupby('Contact Sex')['Profit'].mean().plot(kind='bar')
+
+#Question 2: If business is cash-constrained, which gender should be targeted?
+#The mean cost from females is $751.84
+#while the mean cost for males is $623.76
+
+
+df.groupby('Contact Sex')['Our Cost'].mean()
```

Comment:
This should look at the ratio of profit to total cost to business (including shipping)